src/index.js

```diff
@@ -1,56 +1,21 @@
+import ToDo from './todo.js';
+import addEventsDragAndDrop from './drag-and-drop.js';
+import Task from './task.js';
 import './style.css';
-import Menu from './assets/menu.svg';
-import Refresh from './assets/refresh.svg';
 
-const toDoList = document.getElementById('to-do-list');
-const tasks = [
-  {
-    description: 'Wake up',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Make breakfast',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Write some code',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Submit code review',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Merge pull request',
-    completed: false,
-    index: 4,
-  },
-];
+const toDo = new ToDo();
 
-const refrestContainer = document.getElementById('refreshcontainer');
-const refreshIcon = document.createElement('img');
-refreshIcon.src = Refresh;
-refrestContainer.appendChild(refreshIcon);
+// Add new task
+document.getElementById('task-description').addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    event.preventDefault();
+    const description = document.getElementById('task-description');
+    const index = toDo.localStorageTasks.length;
+    const task = new Task(description.value, index);
+    description.value = '';
```

Comment:
* Kindly implement [These Functionalities ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Add%202%20new,the%20page%20reloads.)as currently none of them is working 🙏